<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>SmsWindow</class>
 <widget class="QDialog" name="SmsWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>@product@ - SMS</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="3" >
    <widget class="QToolButton" name="addContactToolButton" >
     <property name="minimumSize" >
      <size>
       <width>28</width>
       <height>28</height>
      </size>
     </property>
     <property name="toolTip" >
      <string>Click here to add a phone number</string>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="icon" >
      <iconset resource="../../../../../resources/wengophone.qrc" >:/pics/iconbar/add_contact.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="autoRaise" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="4" >
    <widget class="QLabel" name="counterLabel" >
     <property name="autoFillBackground" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>0 /160</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="3" colspan="2" >
    <widget class="QPushButton" name="sendButton" >
     <property name="text" >
      <string>Send</string>
     </property>
     <property name="icon" >
      <iconset resource="../../../../../resources/wengophone.qrc" >:/pics/contact/sms.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2" >
    <widget class="QLineEdit" name="signatureLineEdit" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>7</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip" >
      <string>Enter your signature here</string>
     </property>
     <property name="maxLength" >
      <number>16</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Signature:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5" >
    <widget class="QTextEdit" name="smsText" >
     <property name="acceptRichText" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2" >
    <widget class="QLabel" name="phoneNumberLabel" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>1</hsizetype>
       <vsizetype>5</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Phone number(s):</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <widget class="QComboBox" name="phoneComboBox" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>7</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip" >
      <string>Enter phone number(s) here&lt;br>They must be separated by &amp; or ; or ,</string>
     </property>
     <property name="editable" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>phoneComboBox</tabstop>
  <tabstop>addContactToolButton</tabstop>
  <tabstop>smsText</tabstop>
  <tabstop>signatureLineEdit</tabstop>
  <tabstop>sendButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../../resources/wengophone.qrc" />
 </resources>
 <connections/>
</ui>
